### PR TITLE
Feature/one playable track at time

### PR DESCRIPTION
**Pull request by Bruswei**

### CODE REVIEW

File: h5p-bildetema-words-grid-view/src/components/WordAudio/WordAudio.tsx [hunk 1]
```diff
@@ -29,6 +31,11 @@ export const WordAudio: React.FC<WordAudioProps> = ({ word, textVisible }) => {
       audioElement.pause();
       audioElement.currentTime = 0;
     } else {
+      if (contextAudioRef?.current) {
+        contextAudioRef?.current?.pause();
+        contextAudioRef!.current!.currentTime = 0;
```
Comment (boyum) [inline h5p-bildetema-words-grid-view/src/components/WordAudio/WordAudio.tsx:36]: Are these null checks necessary?

```suggestion
        contextAudioRef.current.pause();
        contextAudioRef.current.currentTime = 0;
```

File: h5p-bildetema-words-topic-image/src/components/TopicImageWordAudio/TopicImageWordAudio.tsx [hunk 2]
```diff
@@ -58,6 +60,11 @@ export const TopicImageWordAudio: React.FC<TopicImageWordAudioProps> = ({
       audioElement.pause();
       audioElement.currentTime = 0;
     } else {
+      if (contextAudioRef?.current) {
+        contextAudioRef?.current?.pause();
+        contextAudioRef!.current!.currentTime = 0;
```
Comment (boyum) [inline h5p-bildetema-words-topic-image/src/components/TopicImageWordAudio/TopicImageWordAudio.tsx:65]:
```suggestion
        contextAudioRef.current.pause();
        contextAudioRef.current.currentTime = 0;
```

File: h5p-bildetema/src/components/TopicGridElementAudio/TopicGridElementAudio.tsx [hunk 3]
```diff
@@ -29,9 +32,13 @@ export const TopicGridElementAudio: React.FC<TopicGridElementAudioProps> = ({
       audioElement.pause();
       audioElement.currentTime = 0;
     } else {
+      if (contextAudioRef?.current) {
+        contextAudioRef?.current?.pause();
+        contextAudioRef!.current!.currentTime = 0;
```
Comment (boyum) [inline h5p-bildetema/src/components/TopicGridElementAudio/TopicGridElementAudio.tsx:37]:
```suggestion
        contextAudioRef.current.pause();
        contextAudioRef.current.currentTime = 0;
```

File: h5p-bildetema-words-grid-view/src/components/WordAudio/WordAudio.tsx (hunk 1 above)
Comment (Bruswei) [reply h5p-bildetema-words-grid-view/src/components/WordAudio/WordAudio.tsx]: You are correct! Removed in all files in newest commit.

**Pull request merged**